go/pkg/pacman.go

```
@@ -2,457 +2,454 @@ package pkg
+	if installed, err := p.InstalledVersion(ctx, name); err != nil {
 		slog.Debug("failed to get installed version", "package", name, "error", err)
 	} else {
 		info.Installed = installed
 	}
 
-	// Get version from sync database
-	out, err := readCmd(p.ctx, "pacman", "-Si", name).Output()
+	out, err := readOut(ctx, p.r, "pacman", "-Si", name)
 	if err != nil {
-		return info, nil
+		return info, nil // not in any sync repo
```

golangci-lint (2.12.2)

[error] error is not nil (line 311) but it returns nil

(nilerr)